### ISSUE TITLE
请问训练自己的数据集为什么LOSS不收敛，结果图都是全黑或者全白。

### ISSUE THREAD

**Issue by xanxuso** [meteorshowers/hed-pytorch#5]
May I ask why the LOSS does not converge when training our own data set, the result graphs are all black or all white.

**Comment by meteorshowers** [comment]
maybe you should change the learning rate.
